ide-support/revsblibrary.livecodescript

```diff
@@ -1857,3 +1864,28 @@ command revSBRemoveInclusions pStack, pInclusions, pConfirm, @xSettings
       end if
    end repeat
 end revSBRemoveInclusions
+
+command revSBListWidgetsOfStack pStack, @xWidgetKindsA
+   local tNumCards, tNumWidgets, tKind
+   put the number of cards of pStack into tNumCards
+   repeat with tCard = 1 to tNumCards
+      put the number of widgets of card tCard of pStack into tNumWidgets
+      repeat with tWidget = 1 to tNumWidgets
+         put the kind of widget tWidget of card tCard of pStack into tKind
+         put "" into xWidgetKindsA[tKind]
```

Comment:
The array isn't actually passed back from the 'list widgets in use' function, I've just used an array as a convenient way of removing duplicates